source/Xunit.Gherkin.Quick.UnitTests/ScenarioExecutorTests.cs

```diff
@@ -349,7 +349,69 @@ public void NonMatchingStep4_after()
             }
         }
 
-        [Fact]
+		[Fact]
+		public async Task ExecuteScenario_Executes_Background_Steps_First()
+		{
+			var gherkinFeaure = new GherkinFeatureBuilder()
+				.WithBackground(sb => sb
+					.Given("given background", null)
+					.When("when background", null)
+					.Then("then background", null))
+				.WithScenario("test scenario", sb => sb
+					.Then("step one", null))
+				.Build();
+
+			var gherkinDocument = new Gherkin.Ast.GherkinDocument(gherkinFeaure, new Gherkin.Ast.Comment[0]);
+
+			_featureFileRepository.Setup(r => r.GetByFilePath($"{nameof(FeatureWithBackgroundSteps)}.feature"))
+				.Returns(new FeatureFile(gherkinDocument))
+				.Verifiable();
```

Comment:
It's good that you made the repository verifiable. To make this really valuable and do something, you should also verify this repository in the assertions section. Like this `_featureFileRepository.Verify()`. That will verify all verifiable setups, including this one.

Reply:
Done